empress/recon_vis/tree.py

```diff
@@ -39,6 +79,11 @@ def __init__(self):
         self.col = None         # float: logical position of this Node in rendering
         self.x = None           # int: x-coordinate for rendering
         self.y = None           # int: y-coordinate for rendering
+        self.upper_v_track = 1  # int: track number for upper vertical host edges
```

Comment:
I prefer the name `lower_vertical_track`, `lower_horizontal_track` but this is up to you.